exercises/bowling/.meta/template.j2

```diff
@@ -0,0 +1,28 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% macro test_case(case) -%}
+    {%- set input = case["input"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        rolls = {{ input["previousRolls"] }}
+        game = self.roll_new_game(rolls)
+        {% if case is error_case -%}
+        with self.assertRaisesWithMessage(BaseException):
```

Comment:
Practically speaking we should never need -- or want -- to catch BaseException ... both IndexError and ValueError are subclasses of Exception, which is itself a subclass of BaseException. Unfortunately things like SystemExit and KeyboardInterrupt are _also_ subclasses of BaseException and we _definitely_ don't want to catch those.